internal/provider/kubernetes/config/default/kustomization.yaml

```diff
@@ -6,7 +6,7 @@ namespace: envoy-gateway-system
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: envoy-gateway-
+# namePrefix: envoy-gateway-
```

Comment:
See my other comments in reference to changing this. Not sure why this change is needed.